source/settings/core.rst

```diff
@@ -3418,19 +3418,15 @@ See :ref:`settings` for list of all setting groups.
 .. dovecot_core:setting:: ssl_options
    :default: no-compression
    :seealso: @ssl;dovecot_core, @dovecot_ssl_configuration
-   :values: compression, no_compression, no_ticket
+   :values: compression, no_ticket
 
    Additional options for SSL.
 
    Currently supported options are:
 
    ``compression``
 
-     (before v2.3) Enable compression.
-
-   ``no_compression``
-
-     (v2.3+) Disable compression.
```

Comment:
Looks like these are wrong in the v2.3 documentation now. We should have a separate fixup commit for that.

Reply:
Done in https://github.com/dovecot/documentation/pull/436